.github/workflows/sync-main-to-prod.yml

```diff
@@ -13,16 +13,26 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-name: Sync Repository with GCS Preprod Bucket
+# This sync will execute when any combination of the following files,
+# and no other files, are changed on the main branch:
+# - timestamp.json
+# - snapshot.json
+# - [0-9]+.snapshot.json
+# Under this condition, the entire main branch (all files) will sync
```

Comment:
nit, the repository/repository folder on main branch, rather than all files

Reply:
Updated